lib/config.coffee

```diff
@@ -230,16 +244,27 @@ exports.getByApplication = (application, options = {}) ->
 # .then (configuration) ->
 # 	console.log(configuration)
 ###
-exports.getByDevice = (uuid, options = {}) ->
+exports.getByDevice = (uuid, deviceApiKey, options = {}) ->
```

Comment:
It's a required field, I had been going to have back-compat in this module but I think it makes sense to only have it in the cli/sdk as they're the real user-facing modules